quicktests/overlaying/tests/realistic/scatter_bar.js

```diff
@@ -25,7 +25,7 @@ function run(svg, data, Plottable) {
   var bars = new Plottable.Plots.Rectangle()
     .addDataset(ds1)
     .x(function (d) { return d.overall; }, xScale)
-    .x2(function(d) { return d.leadership; }, xScale)
+    .x2(function(d) { return d.leadership; })
```

Comment:
The second argument to `x2` is now the `postScale` function; this was breaking the examples